IceCream/Classes/CreamAsset.swift

```diff
@@ -23,12 +23,18 @@ public class CreamAsset: Object {
     override public static func ignoredProperties() -> [String] {
         return ["data", "filePath"]
     }
+    
+    private convenience init(empty: Bool) {
+        self.init()
+        self.data = nil
+    }
 
-    private convenience init(objectID: String, propName: String, data: Data) {
+    private convenience init(objectID: String, propName: String, data: Data, force: Bool = false) {
```

Comment:
Personally I don't think the word `force` promotes clear usage here. We have to come up with a better name. Reference: [API Design Guidelines](https://swift.org/documentation/api-design-guidelines/)